pkgs/applications/graphics/geeqie/default.nix

```diff
@@ -45,7 +48,7 @@ stdenv.mkDerivation rec {
 
     homepage = http://geeqie.sourceforge.net;
```

Comment:
Could you change this to http://geeqie.org?

Reply:
Will do that in 1.4.1. Thank you for your review!